plasmapy/simulation/particle_integrators.py

```diff
@@ -36,6 +36,29 @@ def boris_push(x, v, B, E, q, m, dt):
     dt : `float`
         Timestep, in SI (second) units.
 
+    Examples
+    --------
+    >>> from plasmapy.simulation.particle_integrators import boris_push
+    >>> import numpy as np
+    >>> import matplotlib.pyplot as plt
+    >>>
+    >>> x_0 = np.array([0.0, 0.0, 0.0])
+    >>> v_0 = np.array([5.0, 0.0, 0.0])
+    >>> B = np.array([0.0, 0.0, -5.0])
+    >>> x = []
+    >>> y = []
+    >>>
+    >>> for _ in range(150):
+    >>>     boris_push(x = x_0, v = v_0, B=B, E=0, q=1.0, m=1.0, dt=0.01)
+    >>>     x.append(x_0[0,0])
+    >>>     y.append(x_0[0,1])
+    >>> plt.plot(x,y)
+    >>> plt.xlabel('x [m]')
+    >>> plt.ylabel('y [m]')
+    >>> plt.title(r"Single particle motion without $\vec{E}$ field")
+    >>> plt.grid()
+    >>> plt.gca().set_aspect('equal')
```

Comment:
Rather than plotting, which is not going to display in rendered docs at https://docs.plasmapy.org/en/stable/api/plasmapy.simulation.particle_integrators.boris_push.html#plasmapy.simulation.particle_integrators.boris_push and is best kept in jupyter notebooks such as https://docs.plasmapy.org/en/stable/notebooks/ExB_drift.html, perhaps we can instead display a few examples? This chains into something I also just realized...